.github/workflows/cicd.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main, master ]
```

Comment:
We need to use `main` or `master`, but not both. I do not have a preference, please choose as you wish.